modules/apps/headless/content-space/content-space-headless-apio-api/bnd.bnd

```diff
@@ -0,0 +1,7 @@
+Bundle-Name: Liferay Content Space Headless Apio API
+Bundle-SymbolicName: com.liferay.content.space.headless.apio.api
+Bundle-Version: 2.0.0
```

Comment:
Should be 1

Reply:
This is fixed in the other pull request 👍